alias_service.py

```diff
@@ -0,0 +1,25 @@
+from model import Player
+
+def get_top_suggestion_for_aliases(dao, aliases):
+    return dict( [ (entry[0], min(entry[1], key=lambda p: len(p.name))) for entry in get_player_suggestions_from_player_aliases(dao, aliases).iteritems() ] )
+
+# give suggestions (in the form of Player jsons) for unknown aliases
+def get_player_suggestions_from_player_aliases(dao, aliases):
+    alias_to_suggestion_map = {}
+
+    for alias in aliases:
+        alias_to_suggestion_map[alias] = list(dao.get_players_with_similar_alias(alias))
+
+    return alias_to_suggestion_map
+
+def get_player_or_suggestions_from_player_aliases(dao, aliases):
```

Comment:
Can you throw some quick 1 line comments to describe what these functions do? I'm having a hard time telling them apart.